test/testdata/haddockComments/Record.expected.hs

```diff
@@ -0,0 +1,19 @@
+module Record where
+
+data Record a b c d e f
+  = RecordA
+      {
+-- | 
+a :: a,
+-- | 
+b :: b
```

Comment:
Is this indentation intended? I mean, the `-7` part in the plugin. It looks a bit strange for me...

Reply:
No, it's not intended.  `(1, -7)` looks great when there is only one consturctor, but in this test case it has four. I think dealing with this would be tricky, and we have formatters to do that.

Reply:
Then how about using `(1, 2)`? I believe it will leave fields on the right side of the `{`.

Reply:
Looks great!